pvlib/pvsystem.py

```diff
@@ -46,6 +46,16 @@
 }
 
 
+TEMP_MODELS = {
+    'sapm': {'open_rack_cell_glassback': [-3.47, -.0594, 3],
+             'roof_mount_cell_glassback': [-2.98, -.0471, 1],
+             'open_rack_cell_polymerback': [-3.56, -.0750, 3],
+             'insulated_back_polymerback': [-2.81, -.0455, 0],
+             'open_rack_polymer_thinfilm_steel': [-3.58, -.113, 3],
+             '22x_concentrator_tracker': [-3.23, -.130, 13]},
+    'pvsyst': {'freestanding': (29.0, 0), 'insulated': (15.0, 0)}
```

Comment:
`sapm` models use lists and `pvsyst` models use tuples. Let's be consistent. Perhaps tuples are best for this application but I don't have a strong preference.

Reply:
Good catch, I'd prefer tuples for all

Reply:
My choice for PVSyst was made since these parameters are unchanging so tuples made more sense. 

I'll change all to tuples.